<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManagerWindow</class>
 <widget class="QMainWindow" name="TaskManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mushi</string>
  </property>
  <widget class="QWidget" name="centerWidget">
   <layout class="QVBoxLayout" name="centerWidgetLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTask">
    <property name="title">
     <string>Task</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuTask"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="TaskTreeContainer">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>60</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Tasks</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionDelete"/>
   <addaction name="actionHome"/>
   <addaction name="actionQuickTaskCreator"/>
   <addaction name="actionRefresh"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/images/filenew.png</normaloff>:/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>:/img/error.png</normaloff>:/img/error.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionQuickTaskCreator">
   <property name="icon">
    <iconset>
     <normaloff>:/images/mode_Reference.png</normaloff>:/images/mode_Reference.png</iconset>
   </property>
   <property name="text">
    <string>QuickTask</string>
   </property>
   <property name="toolTip">
    <string>Quick Task</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset>
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normaloff>:/images/mode_Output.png</normaloff>:/images/mode_Output.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
